libs/common/src/services/state.service.ts

```diff
@@ -75,9 +75,13 @@ export class StateService<
     // If the account gets changed, verify the new account is unlocked
     this.activeAccount.subscribe(async (userId) => {
       if (userId == null && this.activeAccountUnlocked.getValue() == false) {
+        this.activeAccountUnlocked.next(false);
         return;
       }
```

Comment:
`activeAccountUnlocked` is already false in this branch, you need a new one

```suggestion
    this.activeAccount.subscribe(async (userId) => {
      if (userId == null && this.activeAccountUnlocked.getValue() == false) {
        return;
      } else if (userId == null) {
        this.activeAccountUnlocked.next(false);
      }
```

Reply:
Ah correct. Although I would expect cryptoMasterKey to be null here.